src/main/java/seedu/address/model/meeting/Meeting.java

```diff
@@ -108,4 +108,22 @@ public String toString() {
                 + "; place: " + place
                 + "; time: " + time.toPrettyString();
     }
+
+    @Override
+    public boolean equals(Object other) {
```

Comment:
I think this essentially does what our updated isSameMeeting() does so its no longer necessary

Reply:
I had to add this method for a test - one of the assert methods calls the equals() method of meeting and it was comparing instances without this implementation